rpcclient/rawtransactions.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/btcsuite/btcutil"
 )
 
+const (
+	// defaultMaxFeeRate is the default maximum fee rate we'll allow for
+	// transactions broadcast to bitcoind v0.19.0 or after nodes.
+	defaultMaxFeeRate = btcutil.SatoshiPerBitcoin / 10
```

Comment:
Why this default? It is also not clear what this number is, is it 10e7 sat/b?

Reply:
This is enforced by `bitcoind`, reworded the comment to note this.